x-pack/plugins/cloud_security_posture/common/types.ts

```diff
@@ -222,3 +222,9 @@ export interface AggFieldBucket {
     doc_count?: string;
   }>;
 }
+
+export type GcpCredentialsType = 'credentials-file' | 'credentials-json';
+
+export type GcpCredentialsTypeFieldMap = {
+  [key in GcpCredentialsType]: string[];
```

Comment:
Can these types be moved above next to the AwsCredentials types, just so it's all located together.